contracts/evm/test/xcall/CallService.t.sol

```diff
@@ -385,6 +439,33 @@ contract CallServiceTest is Test {
         callService.executeCall(1, data);
     }
 
+    function testExecuteCallReply() public {
```

Comment:
Should this test not use the response contract?
And during the execute expect a response to be sent with the defined data?

Reply:
yeah it should but, since helper functions are not added in this, and Response contract follows new message type. Should we proceed with this all as of now, and we can add/change test, after all the pieces of this are merged?
That would be more feasible I guess!

Reply:
Okey i will create a separate issue on testing that we can tackle after to simplify integration testing